crates/solver/src/solver.rs

```diff
@@ -97,6 +97,13 @@ pub enum AuctionResult {
     Rejected(SolverRejectionReason),
 }
 
+pub struct TransactionWithMessage {
+    /// Data needed to reconstruct the simulation
```

Comment:
nit: I guess this comment should rather on the `SimulatedTransaction` struct itself.

Reply:
Moved.